src/phoenix/experimental/evals/functions/binary.py

```diff
@@ -42,18 +38,12 @@ def llm_eval_binary(
         parsed.
     """
 
-    if not (isinstance(template, PromptTemplate) or isinstance(template, str)):
+    if not isinstance(template, PromptTemplate):
         raise TypeError(
-            "Invalid type for argument `template`. Expected a string or PromptTemplate "
+            "Invalid type for argument `template`. Expected a PromptTemplate "
             f"but found {type(template)}."
         )
```

Comment:
Do we still need the explicit type check?

Reply:
i guess it doesn't hurt?